apps/sms/js/threads.js

```diff
@@ -18,6 +18,10 @@
     this.messages = new Map();
   }
 
+  Thread.prototype.getDraft = function() {
```

Comment:
This is my attempt to get rid of confusion about ```hasDrafts``` property that internally made a call to iterate through drafts.